src/Numerics/DGmethods/DGmodel_kernels.jl

```diff
@@ -320,6 +320,9 @@ end
             local_MI[k] = vgeo[ijk, _MI, e]
         end
 
+        # ensure D is loaded
+        @synchronize
```

Comment:
To match the call on [line 438](https://github.com/CliMA/ClimateMachine.jl/blob/97cbf3b357422a56697bd682755e6676a4a65fef/src/Numerics/DGmethods/DGmodel_kernels.jl#L438) do we want

```suggestion
        @synchronize(dim == 3)
```

Reply:
Yes, good catch !